go-controller/pkg/cni/ovs.go

```diff
@@ -98,17 +99,29 @@ func ofctlExec(args ...string) (string, error) {
 	return stdoutStr, nil
 }
 
-func waitForPodFlows(mac string) error {
+func waitForPodFlows(mac string, ipNets []*net.IPNet) error {
```

Comment:
can you please add a TODO comment here that says we need to revisit this when OVN provides a more generic vport readiness API?

Reply:
sure, do we have an RFE open for this already?